mobile/src/main/java/org/openhab/habdroid/ui/WidgetListFragment.kt

```diff
@@ -203,20 +207,32 @@ class WidgetListFragment : Fragment(), WidgetAdapter.ItemClickListener,
     }
 
     override fun onContextItemSelected(item: MenuItem): Boolean {
-        val context = context
+        val activity = activity
```

Comment:
Can this be invoked while the fragment is detached from the activity?
If not I would prefere to requireActvitiy() here.

Reply:
I'm not sure if it might happen due to some bad timing, so I'd like not to throw when `activity == null`.